eng/restore/docs.targets

```diff
@@ -9,7 +9,7 @@
           AfterTargets="Restore">
 
     <ItemGroup>
-      <DocFile Include="$(XmlDocFileRoot)**\*.xml"/>
+      <DocFile Include="$(XmlDocFileRoot)\net\**\*.xml;$(XmlDocFileRoot)\dotnet-plat-ext\**\*.xml"/>
```

Comment:
I tried to include all the files under `$(XmlDocFileRoot)\*\**\*.xml` but it threw errors.

Instead, appending with a semicolon the two paths worked.

Reply:
Talked offline with Viktor, he's ok with this.